### PR TITLE
HTML-Forms

### PR DESCRIPTION
**Pull request by digitekh2i**
In this project, I re-created the "intuit.com" sign up page.
-  Standard label form format
- Sign in and Create Account buttons

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# Status: Changes required 🚨

## Hi @digitekh2i 🖐! Awesome project 🎉✨!

### However, we need to work on a couple of issues before we can move on.

- There are some Stylelint errors. Please fix them. From the next time, remember to fix any Styelint errors before requesting for a code review.

image

- You have this PR from `gh-pages`. This branch is normally reserved for creating Github pages and not working on the features of a project. Remember to create a feature branch that has a descriptive name.

- Change the background color of the clone to match the original.

#### Clone

*background-color: #e9f0ea;*
image

#### Original

*background-color: #eceef1;*
image

- The phone input field must be of type `tel`, eg., `<input type="tel" ...`.

- The `captcha` part is missing.

#### Clone
image

#### Original
Annotation 2020-06-21 171546
Remember to match its background color with the background color of the page.

- The lock icons are missing in the `sign-in` and `create account` buttons.

#### Clone
Annotation 2020-06-21 171813

#### Original
Annotation 2020-06-21 172001

- There are some invalid CSS properties in your CSS file. Be sure to check the comments below.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+background-color: #ffffff;
+border: 2px solid #e2dede;
+margin: auto;
+text-align: center;
+height: 100%;
+padding-top: 2px;
+padding-bottom: 30px;
+margin-top: 30px;
+margin-right: 150;
+margin-right: 150;
```
Comment (tirthajyoti-ghosh) [inline style.css]: Don't forget to add units for your margin values, otherwise, they are deemed invalid by the browser.

**Review by tirthajyoti-ghosh — COMMENTED** [review]
Apply these changes and request for another review.

Happy coding 👨‍💻

**Comment by digitekh2i** [comment]
# changelog
- fixed CSS style errors
- changed the background color to fit the original page
- added reCAPTCHA section
- added lock icon to the buttons
- omitted CSS properties that are not in use

# I created a feature branch titled "dev-source" but while trying to update the pull request to compare master -> dev-source, as opposed to master -> gh-pages. I realized that it was doable. the available solution is to create another pull request that would compare master -> dev-source, but it is against the Microverse-Github Workflow. thus, that's why i am re-applying for code review with gh-pages again. Though I have taken corrections from all the feedback.

**Review by elmejdki — APPROVED** [review]
# Great job @digitekh2i :clap: :clap: you have made a great job :tophat: 
### status: approved

now you can merge to master your project has been approved.

### Happy Coding :smiley:

**Pull request merged**